tests/installation/sle11_releasenotes.pm

```diff
@@ -9,12 +9,20 @@ sub run(){
 
     if (get_var("ADDONS")) {
         if (check_screen 'release-notes-tab') {
+            if (check_var('ADDONS', 'smt')) {
```

Comment:
taking that the code in your else is already splitting ADDONS, I suggest to put your if into that loop